Source/CombatExtended/Compatibility/VanillaPsycastExpanded.cs

```diff
@@ -55,45 +55,45 @@ private static IEnumerable<IEnumerable<IntVec3>> ShieldZones(Thing thing)
 
         private static bool Unsuppresable(Pawn pawn, IntVec3 origin) => pawn.health.hediffSet.hediffs.Any(x => x.GetType() == typeof(Hediff_Overshield));
 
-        private static bool BeforeCollideWith(ProjectileCE projectile, Thing collideWith)
-        {
-            if (collideWith is Pawn pawn)
-            {
-                var interceptor = pawn.health.hediffSet.hediffs.FirstOrDefault(x => x.GetType() == typeof(Hediff_Overshield)) as Hediff_Overshield;
-                if (interceptor != null)
-                {
-                    OnIntercepted(interceptor, projectile, null);
-                    return true;
-                }
-            }
-            return false;
-        }
-        private static bool ImpactSomething(ProjectileCE projectile, Thing launcher)
-        {
-            return Hediff_Overshield_InterceptCheck(projectile, projectile.ExactPosition.ToIntVec3(), launcher);
-        }
-        public static bool Hediff_Overshield_InterceptCheck(ProjectileCE projectile, IntVec3 cell, Thing launcher)
-        {
-            foreach (var interceptor in projectile.Map.thingGrid.ThingsListAt(cell).OfType<Pawn>()
-                .SelectMany(x => x.health.hediffSet.hediffs)
-                .Where(x => x.GetType() == typeof(Hediff_Overshield)).Cast<Hediff_Overshield>())
-            {
-                var def = projectile.def;
-                var result = interceptor.pawn != launcher && (interceptor.pawn.Position == cell);
-                if (result)
-                {
-                    OnIntercepted(interceptor, projectile, null);
-                    return result;
-                }
-            }
-            return false;
-        }
-        public static bool AOE_CheckIntercept(ProjectileCE projectile, Vector3 from, Vector3 newExactPos)
+        //private static bool BeforeCollideWith(ProjectileCE projectile, Thing collideWith)
```

Comment:
This is the "last-chance" check for a shield to interpose before hitting a pawn protected by an overshield.  Unless VPE has changed how their overshields work, I think it's still required.

Reply:
Temporary commented it for test purposes only. Will uncomment it later